hnn_core/cell.py

```diff
@@ -208,6 +208,10 @@ def __repr__(self):
     def L(self):
         return self._L
 
+    @L.setter
+    def L(self, value):
+        self._L = value
```

Comment:
```suggestion
    @L.setter
    def L(self, value):
        self._L = value
```
```suggestion
    @L.setter
    def L(self, value):
        self._L = value
        self.update_end_pts()
```

Reply:
Also, is there a particular reason why we made the `Section.L` attribute private to begin with?

Reply:
It'd be awesome if this worked, but unfortunately `update_end_pts()` is a method of `Cell` and not `Section`.

If this is the path, I think I'll need to go back to section modifications being called from `Cell`, rather than modifying the section directly. That way the end points can be updated every time.

Reply:
I think it makes more sense as a method of `Cell` because you have to modify all the other sections of the cell for it to be consistent ...

Reply:
Oh I see now. That makes sense!